tasks/webhook

```diff
@@ -62,12 +71,62 @@ def setup():
         shutil.copyfile(src, dest)
     os.umask(old_umask)
 
+def ensure_cockpit_checkout():
+    if not os.path.isdir(COCKPIT_CHECKOUT):
+        url = 'https://github.com/cockpit-project/cockpit/'
+        subprocess.check_call(['git', 'clone', url, COCKPIT_CHECKOUT])
+    subprocess.check_call(['git', '-C', COCKPIT_CHECKOUT, 'fetch', 'origin'])
```

Comment:
Wouldn't *this* be a good place for a temporary hack, i. e. checking out your repo/branch instead? Then we can watch the queue get filled and experiment with it, without having to touch the existing runners at all. If you are already past that stage, then ignore this of course.

Reply:
oh this is just for the publishing, the consumers need to have the hack branch too, which is why it needs to be edited in 2 places. Once in the webhook and once in cockpit-tasks